samcli/local/docker/container.py

```diff
@@ -20,7 +20,7 @@
 
 LOG = logging.getLogger(__name__)
 
-START_CONTAINER_TIMEOUT = float(os.environ.get("SAM_CLI_START_CONTAINER_TIMEOUT", 5))
+START_CONTAINER_TIMEOUT = float(os.environ.get("SAM_CLI_START_CONTAINER_TIMEOUT", 10))
```

Comment:
Is there any bad if we just set an even bigger timeout? like 30s?

Reply:
I don't think so, it's just that in my previous tests it didn't take more than like 3 seconds with Rancher Desktop, so I thought this would be safe, but it seems other alternatives might need more time. We do allow to customize this timeout through the env variable but yeah we should provide a good default. Do you suggest 30 secs? Maybe 20?

Reply:
Actually I guess our previous default for invoke was like 3 seconds, so we probably shouldn't make it 10x what this was?

Reply:
If 20s is more than double of the average wait time for colima, let's do 20s.